inference-engine/tests/functional/plugin/cpu/shared_tests_instances/behavior/executable_network/unique_node_names.cpp

```diff
@@ -0,0 +1,55 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vector>
+#include <exec_graph_info.hpp>
+
+#include "behavior/executable_network/unique_node_names.hpp"
+#include "common_test_utils/test_constants.hpp"
+
+using namespace ExecutionGraphTests;
+
+namespace {
+
+TEST_P(ExecGraphUniqueNodeNames, CheckUniqueNodeNames) {
+    InferenceEngine::CNNNetwork cnnNet(fnPtr);
+
+    auto ie = PluginCache::get().ie();
+    auto execNet = ie->LoadNetwork(cnnNet, targetDevice);
+
+    InferenceEngine::CNNNetwork execGraphInfo = execNet.GetExecGraphInfo();
+
+    int numReorders = 0;
+    int expectedReorders = 2;
+    std::unordered_set<std::string> names;
+
+    auto function = execGraphInfo.getFunction();
+    ASSERT_NE(function, nullptr);
+
+    for (const auto & op : function->get_ops()) {
+        const auto & rtInfo = op->get_rt_info();
+        auto it = rtInfo.find(ExecGraphInfoSerialization::LAYER_TYPE);
+        ASSERT_NE(rtInfo.end(), it);
+        auto opType = std::dynamic_pointer_cast<ngraph::VariantImpl<std::string>>(it->second);
+        ASSERT_NE(nullptr, opType);
+
+        if (opType->get() == "Reorder") {
+            numReorders++;
+        }
+    }
+
+    ASSERT_EQ(numReorders, expectedReorders) << "Expected reorders: " << expectedReorders << ", actual reorders: " << numReorders;
```

Comment:
Looks like it is not a purpose of the test, name doesn't have any information about this plugin specific check.

Reply:
Thanks:)